templates/firefox-ios.xliff

```diff
@@ -6278,6 +6282,34 @@ Sent from %2$@ 🦊 %3$@</source>
         <source>Close</source>
         <note>This is the swipe action title for closing an inactive tab by swiping, located in the Inactive Tabs section of the Tabs Tray</note>
       </trans-unit>
+      <trans-unit id="TabTray.TabsSelectorNormalTabsTitle.v140" xml:space="preserve">
+        <source>%@ Tabs</source>
```

Comment:
We still do not handle proper plural support, please read our [documentation](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#add-localization-notes) for more info and going forward.
`Tabs: %@`

Reply:
Tagging @lmarceau

Reply:
Opened a PR to account for this [here](https://github.com/mozilla-mobile/firefox-ios/pull/26586/files), let me know if those changes still needs further improvement